apis/tankerkoenig.js

```diff
@@ -106,8 +106,10 @@ function sortByDistance(a, b) {
  * @returns {boolean} To keep or filter the station.
  */
 function filterStations(station) {
+    const excludeStations = Array.isArray(config.excludeStationIds);
```

Comment:
I would add `excludeStationIds: []` to the module defaults instead, which would also allow to remove one check in the if statement below

Reply:
I guess this then should also be true for stationIds, right?